src/app/auth/page.tsx

```diff
@@ -6,14 +6,16 @@ import { ThemeToggler } from '@/components/ui/theme-toggler'
 
 export default function Auth() {
   return (
-    <AuthContent>
-      <ImageCard className="-my-[4.375rem] hidden sm:flex">
+    <AuthContent className="pt-24">
+      <ImageCard className="-my-[4.375rem] hidden md:flex">
         <Icons.golub />
       </ImageCard>
       <article>
         <header className="flex justify-between pb-2">
```

Comment:
We can remove this padding - maybe I've forgot to remove it previously

```suggestion
        <header className="flex justify-between">
```

Reply:
Ezze, thanks